### ISSUE TITLE
Support setting PYTHONPATH environment variables (or any environment variable) 

### ISSUE THREAD

**Issue by sam-goodwin** [extremeheat/JSPyBridge#114]
I am loading `pythonia` from a node host process.

I ran into a problem where I could not import python code written locally in my code. I can import modules from my virtual environment, but not locally written python code.

To solve, I needed to set PYTHONPATH, but there was no way to do this with the current code.

I worked around it by patching the `IpcPipeCom.js` file to pass through environment variables:

https://github.com/extremeheat/JSPyBridge/blob/ed32525b15f313ad7991b5b8ea697a783fbbad32/src/pythonia/IpcPipeCom.js#L17

```ts
this.proc = cp.spawn(process.env.PYTHON_BIN || 'python3', args, {
  stdio,
  // propagate
  env: process.env
})
```

Then, i can set any environment variables in my node process and have them influence the child python process.

```ts
process.env.PYTHONPATH = rootDir;
```

**Comment by extremeheat** [comment]
Yeah, environment variables set in Node.js with process.env aren't applied to the actual current OS process, but they are treated as such by the Node.js code that checks process.env. As the process' env vars aren't updated, they don't propagate to created child processes, so you do need to pass in the new env vars like you did here.

Feel free to open a PR to do this.
